### PR TITLE
Rename libjpeg to libjpeg-turbo

### PR DESCRIPTION
**Pull request by luhring**

### CODE REVIEW

File: libjpeg-turbo.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+  name: libjpeg-turbo
   version: 2.1.91
   epoch: 1
   description: "Accelerated baseline JPEG compression and decompression library"
   copyright:
     - license: BSD-3-Clause AND IJG AND Zlib
   dependencies:
     provides:
-      - libjpeg-turbo=2.1.91
+      - libjpeg
```
Comment (kaniini) [inline libjpeg-turbo.yaml:10]: If you remove the `libjpeg` provide in the `-dev` package, then you can delete this provide entry too.  Melange will deduce that the two packages are related automatically using SCA.

File: libjpeg-turbo.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 package:
-  name: libjpeg
+  name: libjpeg-turbo
   version: 2.1.91
   epoch: 1
```
Comment (kaniini) [inline libjpeg-turbo.yaml:4]: The epoch should be bumped just to be sure.

**Review by kaniini — CHANGES_REQUESTED** [review]

File: libjpeg-turbo.yaml (hunk 2 above)
Comment (luhring) [reply libjpeg-turbo.yaml]: Done!

File: libjpeg-turbo.yaml (hunk 1 above)
Comment (luhring) [reply libjpeg-turbo.yaml]: Done! (I think!)

**Review by kaniini — APPROVED** [review]

**Pull request merged**